UI/Reports/co/filter_bm.html

```diff
@@ -12,7 +12,7 @@
   <tr>
     <th class="listtop">Balance y Mayor</th>
   </tr>
-  <tr height="5"></tr>
+  <tr style="height:5"></tr>
```

Comment:
This isn't valid content for the style attribute as "5" isn't a valid height (change to "5px" to make it valid); same for all solar occurrences.